cachi2/core/package_managers/yarn/main.py

```diff
@@ -32,6 +32,8 @@ def _resolve_yarn_project(project: Project, output_dir: RootedPath) -> list[Comp
     :param project: the directory to be processed.
     :param output_dir: the directory where the prefetched dependencies will be placed.
     """
+    log.info(f"Fetching the yarn dependencies at the subpath {output_dir.subpath_from_root}")
```

Comment:
nitpick: unrelated to this commit

Reply:
I was wondering if we should start adding the logs on the main functions as the work goes or if we should have a separate story for it. In any case, since we already have the path for the file in that [error message](https://github.com/containerbuildsystem/cachi2/pull/338/files#diff-cbc68c73831a63c3d995893ed401dc07bf92cd8e8654b498177c465e8584ef18), I think I'll just drop this line here.

Reply:
Yeah, just do it in a separate commit please since it's not related to project classes

Reply:
I mean, not add the log line at all in this PR, and leave it for later. Unless you think it's nice to add it now, then I can add it in separate commit.

Reply:
Ah, right. Yeah let's do it now, unless you can make sure that it will, at some point, happen :smile:

Reply:
Ok, kept it in a separate commit.